src/test/java/persistence/study/ReflectionTest.java

```diff
@@ -0,0 +1,100 @@
+package persistence.study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ReflectionTest {
+    private static final Logger logger = LoggerFactory.getLogger(ReflectionTest.class);
+
+    @Test
+    @DisplayName("요구사항1. Car 객체 정보 가져오기")
+    void showClass() {
+        Class<Car> carClass = Car.class;
+        logger.debug(carClass.getName());
+        logger.debug(Arrays.toString(carClass.getDeclaredFields()));
+        logger.debug(Arrays.toString(carClass.getDeclaredConstructors()));
+        logger.debug(Arrays.toString(carClass.getDeclaredMethods()));
+    }
+
+    @Test
+    @DisplayName("요구사항2. test로 시작하는 메소드 실행")
+    void testMethodRun() throws NoSuchMethodException, InvocationTargetException, InstantiationException, IllegalAccessException {
+        Class<Car> carClass = Car.class;
+        Car car = carClass.getDeclaredConstructor().newInstance();
+
+        for (Method declaredMethods : carClass.getDeclaredMethods()) {
+            invokeStartingTestCarInstance(declaredMethods, car);
+        }
+    }
+
+    private static void invokeStartingTestCarInstance(Method declaredMethods, Car car) throws IllegalAccessException, InvocationTargetException {
```

Comment:
```suggestion
    private void invokeStartingTestCarInstance(Method declaredMethods, Car car) throws IllegalAccessException, InvocationTargetException {
```

static 없이 구현되어도 좋을 것 같습니다!